<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="oneButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>One</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoButton">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Two</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeButton">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Three</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>39</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Four</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiveButton">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>39</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Five</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sixButton">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>39</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Six</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sevenButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>68</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Seven</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eightButton">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>68</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eight</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nineButton">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>68</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nine</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
